### ISSUE TITLE
Tags are listed as "Retired in Office" but they are not in the office

### ISSUE THREAD

**Issue by johnpauls** [NCRN/ForestVeg#67]
We need to get an excel spreadsheet with just these tag number and then they can be updated to 'Inactive (Lost)". The spreadsheet can be attached to this issue.

**Comment by lizmatthews03** [comment]
This issue is slightly different from what than the title suggests. We have a suite of Dead Saplings whose Tag Status in the Db is NEITHER Retired in Office NOR Inactive (Lost) -- and all dead saplings should have one or the other tag status.

Courtney went through this list of tags today and cross-walked them with what is here in the lab. We now know which of these sapling tags should be Retired in Office and which should be Inactive (Lost). 

The attached file lists all of the problem Sapling tags and identifies whether or not they are in the office. 

For the tags in the office, Tag Status should be updated to "Retired in Office"
For the tags NOT in the office, Tag Status should be updated to "Inactive (Lost)"

[Dead_SaplingTags_Tag_Numbers updated.xlsx](https://github.com/NCRN/ForestVeg/files/2111242/Dead_SaplingTags_Tag_Numbers.updated.xlsx)

**Comment by blcampbell** [comment]
Tags Courtney listed as RIO (Retired(In Office)) that should be IL (Inactive(Lost)) - 65 of them -- sapling status has various values (Alive Standing, Dead Fallen, Dead Leaning, Dead Missing, Dead Standing). 

@johnpauls, @abrolis --
Is there any reason these sapling statuses would impact the need to adjust the Tag status of these 65 tags to IL?

**Comment by johnpauls** [comment]
@blcampbell , @abrolis 

I think the various "dead" sapling status are fine. Dead sapling tags should either be Retired in Office or Inactive Lost. We don't keep tags on dead saplings like we do for dead trees.

The "Alive Standing" one is an issue, however. That should be active in the field. If a tag was lost and then replaced, would the lost tag still be linked to its old sapling? Otherwise maybe there was a typo somewhere?

**Comment by abrolis** [comment]
I agree with JP on all the dead statuses.  If you could make a list of the tags that are Alive Standing but RIO I can take a look over and see if I can decipher why they were changed.
